data-raw/qc-markdowns/temperature_data_prep.Rmd

```diff
@@ -228,14 +228,14 @@ write_rds(BCK_daily_temps, here::here("data-raw", "standard-format-data-prep","t
 
 ### Recommendations
 
--   CDEC data has very high max - I think these are errors or code for
-    something. CDEC data is more similar to Upper Clear Creek
--   Use the data from Clear over the CDEC data.
+CDEC data has very high max - I think these are errors or code for something. 
+CDEC data is more similar to Upper Clear Creek.
+Use the data from Clear over the CDEC data.
 
 ### CDEC
 
 One station:
-* IGO (1996-09-06 to present H; 1996-05-13 to 2000-01-01 E): TODO ADD DESCRIPTION OF LCOATION
+* IGO (1996-09-06 to present H; 1996-05-13 to 2000-01-01 E): Near Igo, before south folk clear creek confluence. Halfway through the Creek.  
```

Comment:
Good description information. Additionally would be helpful to know how close to RST gage.